mods/ts/sequences/infantry.yaml

```diff
@@ -300,7 +300,6 @@ weedguy:
 		Length: 3
 		ShadowStart: 376
 		Tick: 1000
-	icon: weaticon
```

Comment:
weedguy is not a real actor in TS - it was a hack to make the chemical missile work.  `weaticon` was instead used for the Elite Cadre unit in FS.